File: JMMinistry.Common/Dtos/User/DocumentCheckResultDto.cs
namespace JMMinistry.Common.Dtos.User
{
    public class DocumentCheckResultDto
    {
        public bool Exists { get; set; }
        public bool HasCell { get; set; }
        public string? Name { get; set; }
        public string? LastName { get; set; }
    }
}

File: the_vine_hub_app/lib/features/cells/domain/document_check_result_dto.dart
import 'package:freezed_annotation/freezed_annotation.dart';

part 'document_check_result_dto.freezed.dart';
part 'document_check_result_dto.g.dart';

@freezed
abstract class DocumentCheckResultDto with _$DocumentCheckResultDto {
  const factory DocumentCheckResultDto({
    required bool exists,
    required bool hasCell,
    String? name,
    String? lastName,
  }) = _DocumentCheckResultDto;

  factory DocumentCheckResultDto.fromJson(Map<String, dynamic> json) =>
      _$DocumentCheckResultDtoFromJson(json);
}
